webdriver-spec.html

```diff
@@ -2995,12 +2995,19 @@
  run the following steps:
 
 <ol>
+ <li><p>Start a <var>timer</var>. If this algorithm has not
+  completed before <var>timer</var> reaches
+  the <a>session</a>'s <a>session page load timeout</a> in
+  milliseconds, return an <a>error</a> with <a>error
+  code</a> <a>timeout</a>.
+
  <li><p>If the <a>current session</a>'s <a>page loading strategy</a>
-  is not <code>"none"</code> and if there is an ongoing attempt to
+  is not "<code>none</code>" and if there is an ongoing attempt to
```

Comment:
This should not be inside `<code>` and quotes. Replace it with `<a>none</a>` since it’s a definition in the spec.